src/sage/schemes/elliptic_curves/ell_generic.py

```diff
@@ -2019,6 +2018,30 @@ def division_polynomial(self, m, x=None, two_torsion_multiplicity=2, force_evalu
 
     torsion_polynomial = division_polynomial
 
+    @cached_method
+    def multiplication_by_p_isogeny(self):
```

Comment:
This currently rejects infinite base fields: Should it be moved to `EllipticCurve_finite_field`? Alternatively, have you checked if this works for infinite fields of positive characteristic?

Reply:
Nope, it doesn't work for function fields. I will mark that as a TODO. I guess the 15-year-old issue isn't resolved yet...